nri/index.html

```diff
@@ -1 +1,336 @@
+              <li><a href="#">金融ITソリューション</a></li>
+              <li><a href="#">産業ITソリューション</a></li>
+              <li><a href="#">IT基盤サービス</a></li>
+              <li><a href="#">ソリューション・製品・サービス</a></li>
+              <li><a href="#">お客様事例</a></li>
+              <li><a href="#">未来創発センター</a></li>
+            </ul>
+          </li>
+          <li class="footer-container-list">
+            <a href="#"><strong>サステナビリティ</strong></a>
+            <ul>
+              <li><a href="#">サステナビリティマネジメント</a></li>
+              <li><a href="#">取り組み</a></li>
+              <li><a href="#">ライブラリ</a></li>
+            </ul>
+          </li>
+          <li class="footer-container-list">
+            <a href="#"><strong>ニュース</strong></a>
+            <ul>
+              <li><a href="#">ニュースリリース</a></li>
+              <li><a href="#">お知らせ</a></li>
+              <li><a href="#">イベント</a></li>
+            </ul>
+          </li>
+        </ul>
+      </div>
+      <div class="flexbox-row">
+        <ul>
+          <li class="footer-container-list">
+            <a href="#"><strong>企業情報</strong></a>
+            <ul>
+              <li><a href="#">社長メッセージ</a></li>
+              <li><a href="#">経営方針</a></li>
+              <li><a href="#">NRIの強み</a></li>
+              <li><a href="#">企業案内</a></li>
+              <li><a href="#">国内・海外拠点</a></li>
+            </ul>
+          </li>
+          <li class="footer-container-list">
+            <a href="#"><strong>株主・投資家情報(IR)</strong></a>
+            <ul>
+              <li><a href="#">IRニュース</a></li>
+              <li><a href="#">IRイベント</a></li>
+              <li><a href="#">経営方針</a></li>
+              <li><a href="#">財務・業績</a></li>
+              <li><a href="#">IRライブラリ</a></li>
+              <li><a href="#">株式・格付情報</a></li>
+              <li><a href="#">個人投資家の皆さまへ</a></li>
+            </ul>
+          </li>
+          <li class="footer-container-list">
+            <ul>
+              <li><a href="#"><strong>採用情報</strong></a></li>
+              <li><a href="#"><strong>メディア関係者の皆様へ</strong></a></li>
+              <li><a href="#"><strong>ビジネスパートナーの皆様へ</strong></a></li>
+            </ul>
+          </li>
+          <li class="footer-container-list">
+            <ul>
+              <li><a href="#"><strong>NRI JOURNAL</strong></a></li>
+              <li><a href="#"><strong>NRI Peaple</strong></a></li>
+              <li><a href="#"><strong>NRI Voice</strong></a></li>
+            </ul>
+          </li>
+        </ul>
+      </div>
+      <div class="flexbox-row">
+        <ul>
+          <li class="footer-container-list"><a href="#"><strong>サイトマップ</strong></a></li>
+        </ul>
+      </div>
+      <hr>
+      <div class="site-usage-rules">
+        <ul class="site-usage-rules-list">
+          <li><a href="#">免責条項</a></li>
+          <li><a href="#">サイト利用規定</a></li>
+          <li><a href="#">個人情報保護方針</a></li>
+          <li><a href="#">個人情報の取り扱いについて</a></li>
```

Comment:
```suggestion
          <li><a href="#">個人情報の取扱いについて</a></li>
```
本家だと `個人情報の取扱いについて` ですね。
こういうのはタイピングではなくコピペがよいです。

Reply:
49337b6　修正しました。